priv/static/phoenix.esm.js.map

```diff
@@ -0,0 +1,7 @@
+{
+  "version": 3,
+  "sources": ["../../assets/js/phoenix/utils.js", "../../assets/js/phoenix/constants.js", "../../assets/js/phoenix/push.js", "../../assets/js/phoenix/timer.js", "../../assets/js/phoenix/channel.js", "../../assets/js/phoenix/ajax.js", "../../assets/js/phoenix/longpoll.js", "../../assets/js/phoenix/presence.js", "../../assets/js/phoenix/serializer.js", "../../assets/js/phoenix/socket.js"],
```

Comment:
does esbuild provide a way to modify these paths? Chrome dev tools will parse these paths and group them according to the designated path hierarchy. So in dev tools you'll have to expand `..` then `..` then `assets` then `js` then `phoenix` to finally see the source file listings. If it is possible to re-write these during compilation time to just `phoenix/{filename}` this will be a better dev experience.